client/src/PublicRequest.ts

```diff
@@ -194,6 +207,32 @@ export type SignBtcTransactionRequest
     = SignBtcTransactionRequestStandard
     | SignBtcTransactionRequestCheckout;
 
+export type PolygonTransactionInfo = {
```

Comment:
Yeah, this is a bit of a conundrum. Keyguard only supports USDC, because it only supports sending via our custom transfer contract, which only supports USDC and we only show USDC as the currency ticker in the UI, cannot show anything else.

So while everything runs on Polygon, it can only reasonably interact with USDC. So should we rename all types from Polygon* to Usdc*?

Reply:
The `parseOpenGsnForwardRequest` methods in the sign-polygon-transaction and sign-swap API classes validate that the recipient of any transaction is a known USDC smart contract address. Any other recipient throws. So I think that covers it.